examples/using-gatsby-with-json-yaml/src/pages/404.js

```diff
@@ -0,0 +1,10 @@
+import React from "react"
```

Comment:
I don't think you need to include a 404 page in this example.

Reply:
404 is just added for consistency. My take on this is better leave it here to in order to avoid someone create a issue mentioning that the code does not match the template and subsequently create a pull request to amend it, putting additional work on you, and by you i mean the docs team, that it could be otherwise avoided. Also so that all audiences, ranging from people just getting in touch with Gatsby and Gatsby veterans can be confortable with.